### PR TITLE
Strict linter issues fixes

### PR DESCRIPTION
**Pull request by nickeskov**

### CODE REVIEW

File: pkg/api/api.go [hunk 1]
```diff
@@ -11,32 +11,52 @@ import (
 	"strings"
 	"time"
 
+	"nodemon/pkg/entities"
+	"nodemon/pkg/storing/events"
+	nodesStor "nodemon/pkg/storing/nodes"
```
Comment (alexeykiselev) [inline pkg/api/api.go:16]: I don't get it. Why rename `nodes` to `nodesStor` here? `nodes.Storage` even sounds better than `nodesStor.Storage`.

File: pkg/messaging/pair/server.go [hunk 2]
```diff
@@ -5,30 +5,37 @@ import (
 	"encoding/json"
 	"strings"
 
+	"nodemon/pkg/entities"
+	"nodemon/pkg/storing/events"
```
Comment (alexeykiselev) [inline pkg/messaging/pair/server.go:9]: Why not rename `events` to `eventsStor`?

File: pkg/storing/specific/private_nodes.go [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 import (
 	"sync"
 	"time"
 
-	"github.com/pkg/errors"
-	"go.uber.org/zap"
 	"nodemon/pkg/entities"
 	"nodemon/pkg/storing/events"
-	"nodemon/pkg/storing/nodes"
+	nodesStor "nodemon/pkg/storing/nodes"
```
Comment (alexeykiselev) [inline pkg/storing/specific/private_nodes.go:9]: naming again

**Review by alexeykiselev — APPROVED** [review]
Maybe better not to log zap errors with stdlib logger but panic on them? It happens in almost all mains. And because of that a `zapLogger` package renaming was made.

File: cmd/bots/discord/discord.go [hunk 4]
```diff
@@ -7,94 +7,112 @@ import (
...
+	"nodemon/cmd/bots/internal/common/initial"
 	"nodemon/cmd/bots/internal/common/messaging/pair"
 	"nodemon/cmd/bots/internal/common/messaging/pubsub"
 	"nodemon/cmd/bots/internal/discord/handlers"
-	pairResponses "nodemon/pkg/messaging/pair"
+	pairCommon "nodemon/pkg/messaging/pair"
+
+	"github.com/pkg/errors"
+	"github.com/procyon-projects/chrono"
+	gow "github.com/wavesplatform/gowaves/pkg/util/common"
```
[the remark this answers is not in the document]
Comment (nickeskov) [reply cmd/bots/discord/discord.go:19]: It's `gowaves` package.

**Review by alexeykiselev — APPROVED** [review]

**Review by alexeykiselev — APPROVED** [review]

**Pull request merged**